### PR TITLE
[RFR] Feature/ttf 561 pagination

### PR DESCRIPTION
**Pull request by enginehouse**

### CODE REVIEW

**Comment by enginehouse** [comment]
Right @Koded @openbook, there are a couple of linting issues here: one is a thing where it hates a bit operator and another where there is a space before an object key (I don't think there is : )))

File: app/server/api/v1/api.js [hunk 1]
```diff
@@ -253,15 +256,39 @@ router.get('/news', (req, res, next) => {
...
       response.list = resolveResponse(contentfulResponse)
+      response.list = response.list.map(v => {
+        if (v.fields.originalPublishDate) {
+          v['originalPublishDate'] = v.fields.originalPublishDate
+          v['originalPublishDateFormatted'] = format(Date.parse(v.fields.originalPublishDate), 'Do MMM YYYY')
+        }
+
+        if (v.fields.bodyLegacy) {
+          v.fields.bodyLegacy = _.truncate(v.fields.bodyLegacy, {
+            'length': 100
```
Comment (openbook) [inline app/server/api/v1/api.js]: bodyLegacy is going to be removed - it's in there just for the migration, I think you can use the body field only.

Comment (enginehouse) [reply app/server/api/v1/api.js]: Ok - should I leave it in for testing purposes? Only, with no text coming through Dan will lose it : )

Comment (openbook) [reply app/server/api/v1/api.js]: there's a task for Steve/Laura to migrate all the bodylegacy content into the new field, which I think steve has already done in the case of news items we're keeping.  It doesn't need to block the merge, just thought Id clarify what that was for.

Comment (enginehouse) [reply app/server/api/v1/api.js]: Ahhhh ok - I did not know that - I should have found out : /

**Pull request merged**